### ISSUE TITLE
Avatars display issue

### ISSUE THREAD

**Issue by antonkomarev** [joni-jones/yii2-wschat#3]
I've got an error with avatars.
After sending a message avatar image is always empty, but after page refresh it's displaying correctly.
Is it working well for you?

**Comment by YevSent** [comment]
I will check ASAP this issue and give you answer.

**Comment by YevSent** [comment]
@a-komarev, do you use chat for authenticated users?

**Comment by antonkomarev** [comment]
Yes, and it's taking user data from ActiveRecord model correctly.
Maybe it's because I'm not using common images for avatars? I'm using Gravatar service for it.

**Comment by YevSent** [comment]
Can you post example of response from `Chat.Room.auth()` method?
UPD: the problem with default avatar or with your custom image?
